ee/clickhouse/queries/funnels/test/test_funnel.py

```diff
@@ -744,3 +744,430 @@ def test_funnel_step_conversion_times(self):
+            team=self.team,
+            event="sign up",
+            distinct_id="person2",
+            properties={"key": "val", "$browser": "Safari"},
+            timestamp="2020-01-02T14:00:00Z",
+        )
+        _create_event(
+            team=self.team,
+            event="play movie",
+            distinct_id="person2",
+            properties={"key": "val", "$browser": "Safari"},
+            timestamp="2020-01-02T16:00:00Z",
+        )
+
+        result = funnel.run()
+        self.assertEqual(
+            result[0],
+            [
+                {
+                    "action_id": "sign up",
+                    "name": "sign up",
+                    "order": 0,
+                    "people": [person1.uuid],
+                    "count": 1,
+                    "type": "events",
+                    "average_conversion_time": None,
+                    "breakdown": "Chrome",
+                },
+                {
+                    "action_id": "play movie",
+                    "name": "play movie",
+                    "order": 1,
+                    "people": [person1.uuid],
+                    "count": 1,
+                    "type": "events",
+                    "average_conversion_time": 3600.0,
+                    "breakdown": "Chrome",
+                },
+                {
+                    "action_id": "buy",
+                    "name": "buy",
+                    "order": 2,
+                    "people": [person1.uuid],
+                    "count": 1,
+                    "type": "events",
+                    "average_conversion_time": 7200.0,
+                    "breakdown": "Chrome",
+                },
+            ],
+        )
+        self.assertEqual(
+            result[1],
+            [
+                {
+                    "action_id": "sign up",
+                    "name": "sign up",
+                    "order": 0,
+                    "people": [person2.uuid],
+                    "count": 1,
+                    "type": "events",
+                    "average_conversion_time": None,
+                    "breakdown": "Safari",
+                },
+                {
+                    "action_id": "play movie",
+                    "name": "play movie",
+                    "order": 1,
+                    "people": [person2.uuid],
+                    "count": 1,
+                    "type": "events",
+                    "average_conversion_time": 7200.0,
+                    "breakdown": "Safari",
+                },
+                {
+                    "action_id": "buy",
+                    "name": "buy",
+                    "order": 2,
+                    "people": [],
+                    "count": 0,
+                    "type": "events",
+                    "average_conversion_time": None,
+                    "breakdown": "Safari",
+                },
+            ],
+        )
+
+    # should default to event
+    def test_funnel_step_breakdown_event_no_type(self):
```

Comment:
I think the merge resolution added the other tests here, instead of moving this to `breakdown_cases.py`. (Sorry).

Reply:
oh that's why that conflict was confusing thanks for the catch